core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java

```diff
@@ -146,6 +146,8 @@
   public static final SqlFunction TO_NUMBER = SqlLibraryOperators.TO_NUMBER;
 
   public static final SqlFunction IF = SqlLibraryOperators.IF;
+  public static final SqlFunction RPAD = SqlLibraryOperators.RPAD;
```

Comment:
No need to add operator in SqlStdOperatorTable class

Reply:
removed.